src/Sarif/ToDotNet/ToDotNet.targets

```diff
@@ -6,10 +6,24 @@
   <PropertyGroup>
     <NuGetPath>$(MSBuildProjectDirectory)\..\..\.nuget</NuGetPath>
     <NuGetConfigPath>$(MSBuildProjectDirectory)\..\NuGet.Config</NuGetConfigPath>
-    <JsonSchemaToDotNetPath>$(MSBuildProjectDirectory)\..\packages\Microsoft.Json.Schema.ToDotNet.1.1.0\tools\net461\Microsoft.Json.Schema.ToDotNet.Cli.exe</JsonSchemaToDotNetPath>
     <_ObjectModelOutputDirectory>$(FlavorIndependentIntermediateOutputPath)Autogenerated</_ObjectModelOutputDirectory>
   </PropertyGroup>
-  
+
+  <Choose>
+    <When Condition="$(OS) == 'Windows_NT'">
```

Comment:
how does this work?
why not creating a propertygroup with the condition?

Reply:
you mean remove the <Choose> block , just directly use the 
<PropertyGroup  Condition="$(OS) == 'Windows_NT'">
and
<PropertyGroup  Condition="$(OS) != 'Windows_NT'">
right? 
Sounds good, I will change it.